modules/local/finalize_microbiome_entities.nf

```diff
@@ -0,0 +1,35 @@
+process FINALIZE_MICROBIOME_ENTITIES {
+    label 'process_low'
+
+    conda (params.enable_conda ? "pandas=1.1.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pandas:1.1.5' :
+        'quay.io/biocontainers/pandas:1.1.5' }"
+
+    input:
+    path(entrez_microbiomes_entities)
+    path(nucl_microbiomes_entities)
+    path(microbiomes_entities_noweights)
+    path(entities)
+
+    output:
+    path    "microbiomes_entities.tsv"  , emit: ch_microbiomes_entities  // entity_id, microbiome_id, entity_weight
+    path    "versions.yml"              , emit: versions
+
+    script:
+
+    """
+    finalize_microbiome_entities.py \
+        -eme $entrez_microbiomes_entities \
+        -nme $nucl_microbiomes_entities \
+        -menw $microbiomes_entities_noweights \
+        -ent "$entities" \
+        -o microbiomes_entities.tsv
```

Comment:
Those are typically used with `\\` and not `\`, I was told recently. Maybe you want to change that (here and elsewhere).